stack/app/variables.tf

```diff
@@ -251,6 +251,11 @@ variable "rds_subnet_group_name" {
   type    = string
   default = null
 }
+
+variable "rds_multi_az" {
```

Comment:
should we add to e.g. the description (or the readme), that we default to "true" for "production" ?